pages/common/kafkacat.md

```diff
@@ -0,0 +1,35 @@
+# kafkacat
+
+> Apache Kafka producer and consumer tool.
+
+- Consume messages, start with the newest offset:
+
+`kafkacat -C -t {{topic}} -b {{brokers}}`
+
+- Consume messages starting with the oldest offset and exit after the last message is received:
+
+`kafkacat -C -t {{topic}} -b {{brokers}} -o beginning -e`
+
+- Consume messages as a Kafka consumer group:
+
+`kafkacat -G {{group_id}} {{topic}} -b {{brokers}}`
+
+- Produce message, read from stdin:
+
+` echo {{message}} | kafkacat -P -t {{topic}} -b {{brokers}}`
+
+- Produce messages, read from file:
+
+`kafkacat -P -t {{topic}} -b {{brokers}} {{path/to/file}}`
+
+- List metadata for all topics and brokers:
```

Comment:
Looks like this will list for all topics and a _given_ list of brokers. We should clarify the description to - `List metadata for all topics in a given list of brokers.`

Reply:
Not sure if I understood you correctly, but as far as I can see the command returns all topics and brokers, regardless on what broker I query:

```
./kafkacat -L -b 192.168.1.3:9092
Metadata for all topics (from broker 3: 192.168.1.3:9092/3):
 3 brokers:
  broker 2 at 192.168.1.2:9092
  broker 1 at 192.168.1.1:9092
  broker 3 at 192.168.1.3:9092
 3 topics:
  topic "foo" with 3 partitions:
    partition 2, leader 3, replicas: 3,1, isrs: 1,3
    partition 1, leader 2, replicas: 2,3, isrs: 3,2
    partition 0, leader 1, replicas: 1,2, isrs: 2,1
  topic "bar" with 3 partitions:
    partition 2, leader 1, replicas: 1,2, isrs: 2,1
    partition 1, leader 3, replicas: 3,1, isrs: 1,3
    partition 0, leader 2, replicas: 2,3, isrs: 3,2
  topic "test" with 1 partitions:
    partition 0, leader 1, replicas: 1, isrs: 1
```

I used broker "3" and got back topic "test" that exists only on broker "1"